sdks/go/pkg/beam/pardo.go

```diff
@@ -17,10 +17,9 @@ package beam
 
 import (
 	"fmt"
+	"github.com/apache/beam/sdks/go/pkg/beam/core/graph"
```

Comment:
This doesn't look like a necessary change, and groups the standard library imports with the rest of them. Consider reverting this change.